Doc/reference/compound_stmts.rst

```diff
@@ -199,10 +199,8 @@ returns the list ``[0, 1, 2]``.
 .. versionchanged:: 3.11
    Starred elements are now allowed in the expression list.
 
+
 .. _try:
-.. _except:
-.. _except_star:
-.. _finally:
 
 The :keyword:`!try` statement
```

Comment:
Yeah, I feel you with that sentiment—I was having the same thought over on #96265 . But I also see it would be quite a mouthful, and in those cases, the `try` statement begins the block, with the other statements being optional additions. Up to you if you want me to change it or not.

Reply:
Both alternatives have their cons.  Keeping the current version is probably the best option.